lib/transaction/operation/congress_voting_result.go

```diff
@@ -23,12 +23,14 @@ type CongressVotingResult struct {
 		No    uint64 `json:"no"`
 		ABS   uint64 `json:"abs"`
 	} `json:"result"`
+	CongressVotingHash string `json:"congress_voting_hash"`
```

Comment:
@outsideris CongressVoting and CongressVotingResult operations will be changed by this PR.